xunit-test/QueryTest/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+file(GLOB SRCS *.cpp)
```

Comment:
Should the `QueryTest` changes be in this PR or in https://github.com/infinispan/cpp-client/pull/305?